platform/ios/docs/guides/Info.plist Keys.md

```diff
@@ -19,3 +19,7 @@ The default value is `https://api.mapbox.com`.
 ## MGLMapboxMetricsEnabledSettingShownInApp
 
 If you have implemented custom opt-out of Mapbox Telemetry within the user interface of your app, use this key to disable the built-in check for opt-out support. See [this guide](https://www.mapbox.com/ios-sdk/#telemetry_opt_out) for more details.
+
+## MGLIdeographFontFamilyName
+
+The name of the font family to use for client-side text rendering of CJK ideographs. Set this to the name of a font family which will be available at run time, e.g. `PingFang`.
```

Comment:
iOS and macOS both come with `PingFangHK`, `PingFangSC`, and `PingFangTC`. I wonder which font `PingFang` ends up loading.